nikola/plugins/command/check.py

```diff
@@ -262,7 +252,7 @@ def analyze(self, task, find_sources=False, check_remote=False):
                         self.logger.warn("Broken link in {0}: {1}".format(filename, target))
                         if find_sources:
                             self.logger.warn("Possible sources:")
-                            self.logger.warn("\n".join(_call_nikola_list(self.l, self.site, ["--deps", task])))
+                            self.logger.warn("\n".join(_call_nikola_list(self.site)))
```

Comment:
As I said before, the `doit list` supports a `template` param to format its output (it seems I didnt document this parameter - sorry). Currently the template can only output values for: `name`, `status` and `doc`. I guess the best option would be to extend supported values that can appear in the template. Or maybe support a json output?

There is also the `doit info` command that outputs the targets and dependencies but lacks a formatting and a `--all` options...